rero_ils/modules/cli.py

```diff
@@ -342,10 +346,17 @@ def create(infile, append, reindex, dbcommit, commit, verbose, debug, schema,
         if schema:
             record['$schema'] = schema
         try:
-            rec = record_class.create(record, dbcommit=dbcommit,
-                                      reindex=reindex, pidcheck=pid_check)
-            if append:
-                pids.append(rec.pid)
+            pid = record.get('pid')
```

Comment:
It should be tested if a pid exist in the data. If no pid exist it is a new record.

Reply:
`record_class.record_pid_exists(pid)` will fail if `pid` is not in data.
Anyway I added an extra check for the pid in the if statement.